collections/game-engines/index.md

```diff
@@ -23,7 +23,7 @@ items:
  - WhitestormJS/whs.js
  - wellcaffeinated/PhysicsJS
  - playcanvas/engine
- - cocos2d/cocos2d-html5
+ - cocos/cocos-engine
```

Comment:
Hi, @ipc103 sorry for the late reply, I kinda lost track of it.
I'm trying to do the replacement, because cocos-engine is inheriting all developers from cocos2d-html5 and its own star have also surpassed the legacy engine. With a significant ecosystem growth and more iconic game titles like Topwar, SSSnaker, Monster Land: Builder, Ubisoft Nano etc. I think it's quite fair to do the replacement, so could you please approve it ?

Reply:
Yeah this seems fine to me. I'm going to merge since this has been open for so long.

Reply:
Thanks, and sorry for not responding for a long time